google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -2939,4 +2941,61 @@ public void testBucketLogging() throws ExecutionException, InterruptedException
       RemoteStorageHelper.forceDelete(storage, loggingBucket, 5, TimeUnit.SECONDS);
     }
   }
+
+  @Test
+  public void testUpload() throws Exception {
+    String blobName = "test-upload-static";
+    BlobId blobId = BlobId.of(BUCKET, blobName);
+    byte stringBytes[] = BLOB_STRING_CONTENT.getBytes(UTF_8);
```

Comment:
pull repeated code like this up to a field

Reply:
Fixed with using BLOB_BYTE_CONTENT constant.